### DISCUSSION TITLE
Desktop navigation

### DISCUSSION THREAD

**Discussion by terrakok** [adrielcafe/voyager#30] [General]
Compose Multiplatform has been released: https://www.jetbrains.com/lp/compose-mpp/
It would be nice to support Desktop navigation too.
The Voyager is really nice in using and it has good API! 😃

**Comment by terrakok** [comment]
I've started PoC:
https://github.com/adrielcafe/voyager/pull/33

**Reply by terrakok** [reply]
https://user-images.githubusercontent.com/3532155/145883320-a666d314-1022-4f6c-adfc-ff46c81d3734.mp4

**Comment by adrielcafe** [comment]
Amazing! Thanks again for your contribution @terrakok 🚀

Next steps:
- Migrate other modules to KMM: `voyager-tab-navigator`, `voyager-bottom-sheet-navigator`, `voyager-transitions`, `voyager-kodein`, `voyager-koin`
- Support for Kotlin JS?
- Experimental support for iOS

@programadorthi @DevSrSouza I'll need your help 😉

**Reply by ynsok** [reply]
Any update if JS will be supported? 

@adrielcafe

**Reply by adrielcafe** [reply]
Currently, we're working on iOS support because we're more familiar with it. 

After that, we can try to bring support to JS. If anyone with experience in Kotlin JS decides to help, we appreciate it.
